Dockerfile.backend

```
@@ -10,9 +10,10 @@ FROM ghcr.io/astral-sh/uv:${UV_VERSION} AS uv
 FROM python:${PYTHON_VERSION}-slim AS build
 COPY --from=uv /uv /uvx /bin/
 
-RUN apt-get update && \
-    apt-get install -y --no-install-recommends build-essential libvips-dev pkg-config && \
-    rm -rf /var/lib/apt/lists/*
+RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)